test/solidity/safe-remote-purchase/safe-remote-purchase.js

```diff
@@ -0,0 +1,169 @@
+/*-
+ *
+ * Hedera Smart Contracts
+ *
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+const { expect } = require('chai')
+const { ethers } = require('hardhat')
+
+const contractStates = {
+    Created: 0,
+    Locked: 1,
+    Release: 2,
+    Inactive: 3
+}
+const tinybarToHbarCoef = 100_000_000
+const EVEN_NUMBER_PRICE = 1 * tinybarToHbarCoef
+async function setupContract() {
+    const signers = await ethers.getSigners()
+    const seller = signers[0]
+    const buyer = signers[1]
+    const factoryPurchase = await ethers.getContractFactory('Purchase')
```

Comment:
Please add the "Purchase" keyword to the `constants` file under [Contract object](https://github.com/hashgraph/hedera-smart-contracts/blob/main/test/constants.js#L75) and use the constant variable?